packages/frint-props/README.md

```diff
@@ -297,6 +298,29 @@ const props$ = withState('counter', 'setCounter', 0)(app);
 
 <!-- -->
 
+## withObservable
+
+> withObservable(source$, ...mappers)
+
+### Arguments
+
+1. `source$` (`Observable` OR function returning `Observable`)
+2. `mapper` (`Function`): Returning props OR Observable of props
+
+### Example
+
+```js
+import { of } from 'rxjs/observable/of';
+
+const props$ = withObservable(
+  of({ foo: 'foo value here' }),
+  props => ({ foo: props.foo.toUpperCase() }),
+  props => ({ foo: `${props.foo}!` })
+)();
+```
+
+<!-- -->
```

Comment:
Could you add an example with function-type arguments? It's indeed an important use case.